ITS-UofIowa/herky
# Decide localize functionality

Resolves #31 

How to test locally:
- In an acquia application, checkout this branch (issue-31) in the drush/contrib/herky directory.
- Run `drush herky-localize` (or `hloc` for short) against a local site.
- Review changes.
- Review code at https://github.com/ITS-UofIowa/herky/blob/issue-31/herky.drush.inc#L655 and comment here about what should/should not happen in each environment.
